_data/tools.yml

```diff
@@ -163,6 +163,15 @@
   v2: true
   v3: true
 
+- name: swagger-php
+  category: documentation
```

Comment:
It's not documentation, they just explain that it's a way to generate docs from annotations, but it's really a way too generate specs from annotations, which is something... I personally have concerns about as a workflow in general. 

If we were to add this, we'd need an "Annotations" category, and... @matthewtrask you are potentially less biased here. What do you think?

Reply:
Well, it generates documentation. (json format, you can use with swagger ui)

I really believe it's the best section. (also undocumented usage is to use it to generate documentation by instantiating annotation object, which is nice considering you can mix it with actual annotations)

Reply:
Sorry but I disagree. I could use this to generate mocks so it could be in mocks. It could be used to generate SDK so should it be in SDK?

I think it’s none of them because you’re using annotations to create specs, then any tool to create docs mocks etc. The end goals in a chain of tools does not define that specific tool as being about the end goal. Does that make sense?

Reply:
Well, sdk category then?

Reply:
I feel like you aren't getting my point. 

Annotations > Specs > Docs
Annotations > Specs > Mocks
Annotations > Specs > Contract Tests
Annotations > Specs > SDKs

This tool is Annotations, it is not Specs, and it is not any of the resulting products/artifacts that can be generated from specs.

Reply:
@nek- can you update this PR? would love to get it merged.

Reply:
@matthewtrask please tell me a category. I can add it in misc.

Reply:
@nek- looks like Phil was thinking Annotations above.

Reply:
Sorry to barge into this conversation, IMO tools like Swagger-php, Springfox, Swashbuckle, etc. are solving a different problem than the majority of tools currently listed. These tools are used to _produce_ api specifications. I think the majority of the tools listed on the site _consume_ api specifications.

This distinction has major implications on the development process, e.g. API-First vs. Code-First. Would these types of tools have any use to development teams following an API-First approach?

TL;DR; I think swagger-php and the like deserve their own category such as **Specification Generators**

Reply:
Let's do it! A "Specification Generators" category and this can be the first tool in the category.